src/components/common/Button/Toast.jsx

```diff
@@ -0,0 +1,9 @@
+import styles from './Toast.module.css';
+
+export default function Toast(props) {
```

Comment:
```suggestion
export default function Toast({pop}) {
```
요건 이렇게 구조분해 할당해서 사용해 주면 좋겠네요!